app_base.htm

```diff
@@ -0,0 +1,56 @@
+<%inherit file="/homepage/templates/base.htm" />
```

Comment:
Having comments in your code to help explain what's going on can be really helpful.

Reply:
I'd agree with this, especially because I didn't do that.

Reply:
Thank you so much for the suggestion. I will add comments to my code shortly.